src/workerd/io/worker-interface.h

```diff
@@ -130,6 +131,12 @@ class WorkerInterface: public kj::HttpService {
     // what types.
     virtual uint16_t getType() = 0;
 
+    // Get event info for tracing.
+    // Return none if this event type doesn't need tracing.
```

Comment:
This comment is misleading – for events that are not custom events, we generally do need tracing, but manually enable it instead of using this mechanism (at least based on what this PR is currently set to do).